arrows/core/associate_detections_to_tracks_threshold.cxx

```diff
@@ -102,7 +100,7 @@ ::check_configuration( VITAL_UNUSED vital::config_block_sptr config ) const
 bool
 associate_detections_to_tracks_threshold
 ::associate( kwiver::vital::timestamp ts,
-             kwiver::vital::image_container_sptr /*image*/,
+             kwiver::vital::image_container_sptr, // image
```

Comment:
```suggestion
             kwiver::vital::image_container_sptr /*image*/,
```

Use of inline comments to name "unnamed" parameters is a common idiom that I don't think we should be changing. Also, moving the comma makes for an inconsistency with e.g. `int /*foo*/, int, // bar`, as well as just plain making it less clear that the comment is an "elided" name of a parameter.

I'll repeat something I've said multiple times; I am not convinced that widespread replacement of ***non-Doxygen*** traditional C comments with C99/C++ style is desirable.

Reply:
I've gone back over the files and fixed this back to the original syntax where I could find it in this PR.  I'll take a look at the other PRs as well to see if I can find other places to fix.